draft-ietf-mls-protocol.md

```diff
@@ -906,7 +933,7 @@ key of the non-updated child and the ephemeral public key
 transmitted in the message.
 
 
-## Key Schedule
+## Key Schedule {#key-schedule}
```

Comment:
This tag isn't necessary.  Kramdown automatically makes a tag of this character from the content of the heading.

Reply:
Fixed, Thanks !